main.js

```diff
@@ -165,10 +165,13 @@ function gameWin() {
   }
   console.log(`And the winner is... ${gameWinner}!!!`);
 
+  // Call function to populate winning card
+  addWinCard();
+
   // Create game winning card with players info
   function addWinCard() {
     var winCard = `<section class="game-card">
-      <p class="game-header"><span class="challenger-vs">${challengerOne}</span>vs<span class="challenger-vs">${challengerTwo}</span></p>
+      <p class="game-header"><span class="challenger-vs">${challengerOne.value}</span>vs<span class="challenger-vs">${challengerTwo.value}</span></p>
       <p class="winner-name">${gameWinner}</p>
       <p class="winner-statement">Winner</p>
       <section class="game-footer">
```

Comment:
I think this can be stored globally to keep our function shorter.  The rubric says our functions should be no more than ten lines, and when I met with Will about balancing act, he said it's a good idea to keep those at the top with query selectors. Maybe storing it as winCardHTML would help us differentiate from the div/section as well.

Reply:
I didn't know that is a suggestion of keeping functions shorter. Good to know! I'll make that update and repush! Thanks!

Reply:
JK! Taking that html global broke the game since it relied on the game winner to be determined. So I moved it back to the function. We can potentially refactor this later on.

Reply:
Sounds good. Let's get everything working and then we can refactor at the end to make everything more concise.